trajectory_tracker/src/trajectory_tracker.cpp

```diff
@@ -401,55 +289,24 @@ void TrackerNode::control()
     return;
   }
 
-  float min_dist_ = 10000.0;
-  int iclose = -1;
-  geometry_msgs::Point origin;
-  origin.x = cos(w_ * look_forward_ / 2.0) * v_ * look_forward_;
-  origin.y = sin(w_ * look_forward_ / 2.0) * v_ * look_forward_;
+  const Eigen::Vector2d origin =
+      Eigen::Vector2d(std::cos(w_lim_.get()), std::sin(w_lim_.get())) *
+      (look_forward_ / 2.0) * v_lim_.get() * look_forward_;
```

Comment:
wrong: equivalent to `origin.x = cos(w_) * look_forward_ / 2.0 * v_ * look_forward_;`